sysdig/common_test.go

```diff
@@ -1,5 +1,3 @@
-//go:build tf_acc_sysdig || tf_acc_sysdig_monitor || tf_acc_sysdig_secure || tf_acc_ibm || tf_acc_ibm_monitor || tf_acc_policies || unit
-
 package sysdig_test
```

Comment:
Completely removing build tags will result in always building this file which we want.

Reply:
I'm not familiar with the build & test workflow of the provider, so take this comment with a grain of salt...
The side effect of completely removing this build directive will be that `common_test` will be included in any build, including release builds.

A different approach could be to set a build directive like
```
//go:build !release
```

Reply:
AFIK go build command excludes all *_.test.go files so test files will never be included in final binary. There is no problem here actually.

Reply:
Tags in tests are used to differentiate tests by nature, there are no connection with actual release.